### PR TITLE
Use bulk scan product app insights and update web app module properties

### PR DESCRIPTION
**Pull request by nitinprabhu**
- Updated web app module to use bulk scan app insights.

- Updated other properties such as location, resource provider version, SKU, resource group, app service plan.

-  Needs https://github.com/hmcts/bulk-scan-payment-processor/pull/5 to be merged

### CODE REVIEW

File: infrastructure/main.tf [hunk 1]
```diff
@@ -1,21 +1,29 @@
...
+}
 
 locals {
-  ase_name            = "${data.terraform_remote_state.core_apps_compute.ase_name[0]}"
-  is_preview          = "${(var.env == "preview" || var.env == "spreview")}"
-  local_env           = "${local.is_preview ? "aat" : var.env}"
+  ase_name   = "${data.terraform_remote_state.core_apps_compute.ase_name[0]}"
+  is_preview = "${(var.env == "preview" || var.env == "spreview")}"
+  local_env  = "${local.is_preview ? "aat" : var.env}"
+  sku_size   = "${var.env == "prod" || var.env == "sprod" || var.env == "aat" ? "I2" : "I1"}"
```
Comment (doncem) [inline infrastructure/main.tf]: non of `s` prefixed environments are needed. previous version was good enough. `spreview` also not needed

Comment (doncem) [reply infrastructure/main.tf]: on the other hand, are those (`is_preview` and `local_env`) used anywhere?

Comment (nitinprabhu) [reply infrastructure/main.tf]: good catch removed them 😇

**Comment by nitinprabhu** [comment]
Not needed anymore as we are going straight to AKS

**Pull request closed without merging**